Doc/library/socket.rst

```diff
@@ -425,7 +425,8 @@ Constants
    .. versionchanged:: 3.12
       Added ``SO_RTABLE`` and ``SO_USER_COOKIE``. On OpenBSD
       and FreeBSD respectively those constants can be used in the same way that
-      ``SO_MARK`` is used on Linux.
+      ``SO_MARK`` is used on Linux. Also added missing ``TCP_*`` available in
```

Comment:
as annoying as it is... rather than assume people understand what Linux 6.0 implies, it is nicer for documentation to list all of the new constant names. that way people searching the docs for a name can find it.